src/parser/parsers/heroes.py

```diff
@@ -107,57 +107,19 @@ def _parse_hero_weapon(self, hero_value):
             'ReloadMovespeed': int(weapon_prim['m_flReloadMoveSpeed']) / 10000,
             'ReloadDelay': weapon_prim['m_flReloadSingleBulletsInitialDelay'] if 'm_flReloadSingleBulletsInitialDelay' in weapon_prim else 0,
             'ReloadSingle': weapon_prim['m_bReloadSingleBullets'] if 'm_bReloadSingleBullets' in weapon_prim else False,
-            'BulletSpeed': self.calc_bullet_velocity(weapon_prim['m_BulletSpeedCurve']['m_spline']),
-            'FalloffStartRange': weapon_prim['m_flDamageFalloffStartRange'] / ENGINE_UNITS_PER_METER,
-            'FalloffEndRange': weapon_prim['m_flDamageFalloffEndRange'] / ENGINE_UNITS_PER_METER,
+            'BulletSpeed': calc_bullet_velocity(weapon_prim['m_BulletSpeedCurve']['m_spline']),
+            'FalloffStartRange': round(weapon_prim['m_flDamageFalloffStartRange'] / ENGINE_UNITS_PER_METER,0),
+            'FalloffEndRange': round(weapon_prim['m_flDamageFalloffEndRange'] / ENGINE_UNITS_PER_METER,0),
             'FalloffStartScale': weapon_prim['m_flDamageFalloffStartScale'],
             'FalloffEndScale': weapon_prim['m_flDamageFalloffEndScale'],
             'FalloffBias': weapon_prim['m_flDamageFalloffBias'],
             'BulletGravityScale': weapon_prim['m_flBulletGravityScale'],
-            'BulletRadius': weapon_prim['m_flBulletRadius'] / ENGINE_UNITS_PER_METER,
+            'BulletRadius': round(weapon_prim['m_flBulletRadius'] / ENGINE_UNITS_PER_METER,0),
```

Comment:
looks like this is flattening every value to 0. Maybe 2/3 decimal places here? Also is this radius visible anywhere in game to make comparisons